src/app/component/header/header.component.html

```diff
@@ -116,17 +100,13 @@ <h1>Jyoti Technosoft LLP</h1>
 
 <div
   class="modal"
```

Comment:
Please remove this model to and use model service for this

Reply:
ok I have done Remove model